scene/gui/tabs.cpp

```diff
@@ -241,13 +241,14 @@ void Tabs::_shape(int p_tab) {
 
 	tabs.write[p_tab].xl_text = atr(tabs[p_tab].text);
 	tabs.write[p_tab].text_buf->clear();
+	tabs.write[p_tab].text_buf.instantiate();
 	if (tabs[p_tab].text_direction == Control::TEXT_DIRECTION_INHERITED) {
 		tabs.write[p_tab].text_buf->set_direction(is_layout_rtl() ? TextServer::DIRECTION_RTL : TextServer::DIRECTION_LTR);
 	} else {
 		tabs.write[p_tab].text_buf->set_direction((TextServer::Direction)tabs[p_tab].text_direction);
 	}
 
-	tabs.write[p_tab].text_buf->add_string(tabs.write[p_tab].xl_text, font, font_size, tabs[p_tab].opentype_features, (tabs[p_tab].language != "") ? tabs[p_tab].language : TranslationServer::get_singleton()->get_tool_locale());
```

Comment:
In a long statement like this with ternary operator, I think the parentheses did make sense to clarify the precedence and that `tabs[p_tab].language != ""` is a boolean condition.

If you really want to remove them, then I'd replace `x != ""` by `!x.is_empty()`.

Reply:
Change reverted.